Add paraview docs

holly-t opened the pull request.

kevinstratford reviewed (changes requested):
1. Paraview might want to go in "Data analysis and tools" section rather than "Research software" (or both?)
2. Might be worth moving the 'srun' step before the 'ssh' step. The srun pvserver step  reports both the node id and the port number (could possibly not be 11111...) required for ssh.
3. Might be worth stressing that it is "localhost" from the local client.

holly-t commented:
@kevinstratford I've moved it to the "Data analysis and tools" section. I've also reordered the srun and ssh steps as you suggested and added a note on the localhost.
